apps/src/studio/studio.js

```diff
@@ -3900,13 +3962,27 @@ Studio.endGame = function(opts) {
 
   if (winValue == "win") {
     Studio.trackedBehavior.hasWonGame = true;
-    Studio.setScoreText({text: studioMsg.winMessage()});
+    Studio.setVictoryText({text: studioMsg.winMessage()});
   } else if (winValue== "lose") {
     Studio.trackedBehavior.hasLostGame = true;
-    Studio.setScoreText({text: studioMsg.loseMessage()});
+    Studio.setVictoryText({text: studioMsg.loseMessage()});
   } else {
     throw new RangeError("Incorrect parameter: " + opts.value);
   }
+
+  Studio.gameState = Studio.GameStates.OVER;
+};
+
+Studio.onMouseDown = function (e) {
+  if (Studio.gameState === Studio.GameStates.WAITING) {
+    Studio.runButtonClick();
+  } else if (Studio.gameState === Studio.GameStates.OVER) {
+    // do a reset
+    var resetButton = document.getElementById('resetButton');
```

Comment:
We should consider what will happen if we don't show the Run/Reset button - as already is the case for applab in share mode. I *think* this approach will still work fine, but we may want to call an actual method instead of relying on the button event handlers.